Update .NET supported versions

twsouthwick opened the pull request:
The supported versions of .NET for the package is .NET 4.5.2 and .NET Standard 1.4. By listing only .NET Core, it implies that only .NET Core is supported. This updates the documentation to reflect the actual support of the library.

azurecla commented:
Hi __@AnuragMehrotra__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (anuragm). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

deneha commented:
@AnuragMehrotra Thank you for this contribution.  I'll ask the author to review.

@Jim-Parker Would you please review this change and let me know if I should merge it?

Jim-Parker commented:
@AnuragMehrotra  In the paragraph just above the change, it mentions that the "step is to specify the protection method using the Set-DPMProtectionType cmdlet." Should there be additional info to support the addition of the Add-DPMChildDatasource cmdlet?

AnuragMehrotra commented:
I have added some details to explain the new cmdlet.

Jim-Parker commented:
@AnuragMehrotra added content looks good. Thanks.

twsouthwick committed:
Update .NET supported versions

The supported versions of .NET for the package is .NET 4.5.2 and .NET Standard 1.4. By listing only .NET Core, it implies that only .NET Core is supported. This updates the documentation to reflect the actual support of the library.

twsouthwick committed:
Remove c# from .NET/C# library
